### DISCUSSION TITLE
请问模板编辑器里自带的是什么啊，怎么删掉

### DISCUSSION THREAD

**Discussion by Zane-Zzx** [windingwind/zotero-better-notes#802] [Q&A] [answered]
![image](https://github.com/windingwind/zotero-better-notes/assets/75843496/3276bb7e-39d3-454d-8ced-8399bbff2a0f)

**Comment by windingwind** [comment, marked as answer]
这是某个旧版本在升级时导致的问题。升级bn即可解决；如果升级无法解决，备份需要的模板后，打开设置-高级-设置编辑器-搜索`extensions.zotero.Knowledge4Zotero.templateKeys`，清空即可。

**Reply by Zane-Zzx** [reply]
使用第二个方法成功了，谢谢大佬
